envolve-backend/src/main/java/de/neuefische/erbay/envolve/service/SchoolClassService.java

```diff
@@ -0,0 +1,43 @@
+package de.neuefische.erbay.envolve.service;
+
+import de.neuefische.erbay.envolve.db.SchoolClassDb;
+import de.neuefische.erbay.envolve.model.SchoolClass;
+import de.neuefische.erbay.envolve.model.Student;
+import de.neuefische.erbay.envolve.model.dto.AddSchoolClassDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Service
+public class SchoolClassService {
+
+    private final SchoolClassDb schoolClassDb;
+
+    @Autowired
+    public SchoolClassService(SchoolClassDb schoolClassDb) {
+        this.schoolClassDb = schoolClassDb;
+    }
+
+    public void addClass(AddSchoolClassDto tempSchoolClass, String teacherName) {
+     SchoolClass schoolClass = new SchoolClass();
+     List<Student> tempList = new ArrayList<>();
+     schoolClass.setTeacher(teacherName);
+        for (int i = 0; i < tempSchoolClass.getClassmembers().size(); i++) {
```

Comment:
Created Variable studentnames but Change of Loop to (String name : studentNames) {..} didnt worked out because of "            String singleClassmemberName = tempSchoolClass.getClassmembers().**get(i);**" ...